config.toml.example

```diff
@@ -306,9 +306,15 @@
 #
 #debug = false
 
+# This will make your build more parallel; it costs a bit of runtime
+# performance perhaps (less inlining) but it's worth it.
```

Comment:
This is the `0` value, not any other specific value. IMO this only makes sense on `rustc-dev-guide`. I don't see why the two must have *exactly identical* comments, given the `rustc-dev-guide` *suggests specific values*.